lib/stdlib/test/uri_string_SUITE.erl

```diff
@@ -825,6 +829,59 @@ parse_recompose_autogen(_Config) ->
     Tests = generate_test_vectors(uri_combinations()),
     lists:map(fun run_test_parse_recompose/1, Tests).
 
+resolve_normal_examples(_Config) ->
+    BaseURI = <<"http://a/b/c/d;p?q">>,
+    <<"g:h">> = uri_string:resolve(<<"g:h">>, BaseURI),
+    <<"http://a/b/c/g">> = uri_string:resolve(<<"g">>, BaseURI),
+    <<"http://a/b/c/g">> = uri_string:resolve(<<"./g">>, BaseURI),
+    <<"http://a/b/c/g/">> = uri_string:resolve(<<"g/">>, BaseURI),
+    <<"http://a/g">> = uri_string:resolve(<<"/g">>, BaseURI),
+    <<"http://g">> = uri_string:resolve(<<"//g">>, BaseURI),
```

Comment:
This specific example is not normalized by scheme in RFC3986 and I think that even sheme-based normalization should have been applied here to avoid ambiguity as all other exampes are normalized. I think the correct result is <<"http://g/">>.

Reply:
There doesn't seem to be an errata for this one and the algorithm does not mention normalization (which may be applied afterwards), I just used `normalize` because I didn't have a `remove_dot_segments` in Cowlib. But there's probably one in `uri_string` so I probably just need to swap that `normalize` call with `remove_dot_segments`.

Reply:
Now using the correct function.